src/js/views/patients/shared/components/owner_component.js

```diff
@@ -71,9 +72,17 @@ export default Droplist.extend({
       return {
         collection: getGroupClinicians(group),
         headingText: group.get('name'),
+        className: 'picklist__group',
```

Comment:
I'm pretty sure we don't need to add `picklist__group` anywhere now as it's the default?